.lint

```diff
@@ -0,0 +1,3 @@
+exclusions:
+ template-job-rule:
+ panel-job-instance-rule:
```

Comment:
Can we have a follow-up issue to remove those exclusions?

Reply:
You are right, having `job` and `instance` won't be possible right now for all dashboards.